docs(release-notes): refine 2026-05-15 release notes (#2663)
lmc-eu/spirit-design-system: docs/release-notes/2026-05-15.md

Reviewer: 🤔 Hmm, it should be stored here: https://github.com/alma-oss/spirit-design-system/tree/main/docs/release-notes. Maybe I haven't updated the skill? 🤔

Author: I move it, but the skills need to be updated.

# 2026-05-15 Spirit Design System Release Notes

📢 🚀 🎉 Ahoy, we have a new release with features and improvements. Here we go.

## General Changes

We've rolled out a **rem-based sizing foundation** across design tokens, `@alma-oss/spirit-web`, and `@alma-oss/spirit-web-react`, so spacing and dimensions scale more naturally with user font settings. If you consume tokens or component styles directly, review layouts after upgrading — values that were expressed in `px` now use `rem`.

> We've added **`I18nProvider`** in Web React so you can supply merged translations for localized component copy. **`Picker`** gets richer examples, and we fixed **`ToastContext`** export plus style-utility specificity so custom overrides behave more predictably.

Reviewer 2: ?

```suggestion
We've added **`I18nProvider`** in Web React so you can supply merged translations for localized component copy. **`UNSTABLE_Picker`** gets richer examples, and we fixed **`ToastContext`** export.
```

> We've **deprecated `FileUploader`** in both `@alma-oss/spirit-web` and `@alma-oss/spirit-web-react` — migrate to `UNSTABLE_FileUpload` instead.

Reviewer: suggestion: Maybe add a link to the migration guide?

## News in Packages

> &nbsp;

Reviewer 2: Extra non-breaking space.

Applies for the whole file.

📦 **Web React _4.7.0_** `@alma-oss/spirit-web-react`

### Features

- Add `I18nProvider` and translations #DS-2210 ([7a8d1fe](https://github.com/alma-oss/spirit-design-system/commit/7a8d1fe9c14f4c4338dd4eaf0337c7973a02d592))
- Introduce unstable `Table` component ([2be3ceb](https://github.com/alma-oss/spirit-design-system/commit/2be3ceb47b3ded6f061bcec0c820cb5cadf2d558))
- Replace px with rem #DS-2392 ([ecd927e](https://github.com/alma-oss/spirit-design-system/commit/ecd927e9c5c6fc6ba54540453b2701c55bba682b))
- Deprecate `FileUploader` component and subcomponents #DS-2444 ([9b91bf5](https://github.com/alma-oss/spirit-design-system/commit/9b91bf589e49c0d0c274ea90b811e5e1a5653c15))

### Bug Fixes

> - Fix type for containerProps #DS-2561 ([99c34aa](https://github.com/alma-oss/spirit-design-system/commit/99c34aa1306645b049266667766dd0ea003ee128))

Reviewer: ```suggestion
- Fix type for **containerProps** #DS-2561 ([99c34aa](https://github.com/alma-oss/spirit-design-system/commit/99c34aa1306645b049266667766dd0ea003ee128))
```

- Fix subpath type declarations not resolving for package exports #DS-2561 ([b2d6aaf](https://github.com/alma-oss/spirit-design-system/commit/b2d6aaf4b99d9a2cefeaea18dc16aaad9fb61601))
- Add disabled class for `ControlButton` when `UNSTABLE_File` is disabled ([a535c53](https://github.com/alma-oss/spirit-design-system/commit/a535c534ed7462fc33387b5982cd3fbf8a2c2f8c))
- Export `ToastContext` ([d061e48](https://github.com/alma-oss/spirit-design-system/commit/d061e48e5069f877c561cddb38ac48b43c961911))

> - Extend `Table` demos to cover all web demos ([bfb2245](https://github.com/alma-oss/spirit-design-system/commit/bfb2245dc291596613dea835c17e340ca6985db7))

Reviewer: suggestion: This is more about documenting than fixing a bug. Please, remove.

> - Fix invalid Figma Code Connect for various components ([0bf4d87](https://github.com/alma-oss/spirit-design-system/commit/0bf4d87e3f2c1a0a5c81d4bfc561ef2606869e48))

Reviewer: suggestion: The same as above.

### Features

- Introduce unstable `Table` component ([bf13df1](https://github.com/alma-oss/spirit-design-system/commit/bf13df1da1594897786203f058c43fade40a57de))
- Replace px with rem #DS-2392 ([4fb76e8](https://github.com/alma-oss/spirit-design-system/commit/4fb76e868ae68740dab2338c9fb8293f87092bd8))
- Deprecate `FileUploader` component and subcomponents #DS-2444 ([7086646](https://github.com/alma-oss/spirit-design-system/commit/7086646502f3742987352538165166a7446292f9))

### Bug Fixes

- Fix icon in File upload success validation text ([ca9fb28](https://github.com/alma-oss/spirit-design-system/commit/ca9fb28068410fefe656419b7f281f4feb73a6b3))

[Full changelog](https://github.com/alma-oss/spirit-design-system/compare/@alma-oss/spirit-web@4.4.0...@alma-oss/spirit-web@4.6.0)

&nbsp;

📦 **Design Tokens _4.2.0_** `@alma-oss/spirit-design-tokens`

### Features

- Update tokens to rem #DS-2392 ([cb72cc3](https://github.com/alma-oss/spirit-design-system/commit/cb72cc39b51caa8f8a4ff207f299741bfee30ac6))

[Full changelog](https://github.com/alma-oss/spirit-design-system/compare/@alma-oss/spirit-design-tokens@4.1.2...@alma-oss/spirit-design-tokens@4.2.0)

## What's Next 🔮

> 

Reviewer 2: ?

```suggestion

The release of the new major version is slowly but surely approaching, and we are currently preparing the following features:

```

> - **UNSTABLE_Combobox / ComboboxPicker**

Reviewer 2: ```suggestion
- **UNSTABLE_Combobox**
```

>   New multi-select combobox with inline filtering for search forms (e.g. job positions, languages, locations), building on the Combobox prototype and web implementation.

Reviewer 2: ?

```suggestion
  New multi-select combobox with allowing users to filter a list of options (e.g. job positions or locations) by typing.
```
